api/invite.go

```diff
@@ -464,14 +447,13 @@ func (a *Api) AcceptTeamInvite(res http.ResponseWriter, req *http.Request, vars
 	validationErrors := []error{}
 
 	conf.ValidateStatus(models.StatusPending, &validationErrors).
-		ValidateUserID(userID, &validationErrors).
-		ValidateTeamID(teamID, &validationErrors)
-
-	if conf.Role != "patient" {
-		conf.ValidateType(models.TypeMedicalTeamInvite, &validationErrors)
-	} else {
-		conf.ValidateType(models.TypeMedicalTeamPatientInvite, &validationErrors)
-	}
+		ValidateType([]models.Type{
+			models.TypeMedicalTeamDoAdmin,
+			models.TypeMedicalTeamRemove,
+			models.TypeMedicalTeamInvite,
+			models.TypeMedicalTeamPatientInvite,
+		}, &validationErrors).
+		ValidateUserID(inviteeID, &validationErrors)
```

Comment:
We don't have to validate teamID any more ? 
( `ValidateTeamID(teamID, &validationErrors)` has been removed).

In that case ValidateTeamID method is not used anywhere.

Reply:
My point was also that we don't check anymore that conf.TeamID is not nil (or accept.TeamID not sure if we have to check that in the payload or in the db response).